protozfits/__init__.py

```diff
@@ -231,11 +212,44 @@ def nt(m):
             enum_types[(m, field.name)] = enum
 
 
-def rewind_table():
-    # rawzfitsreader.rewindTable() has a bug at the moment,
-    # it always throws a SystemError
-    # we let that one pass
-    try:
-        rawzfitsreader.rewindTable()
-    except SystemError:
-        pass
+class MultiZFitsFiles:
+
+    def __init__(self, paths):
+        self._files = {}
```

Comment:
In principle this is a valid point, but the `MultiZFitsFiles` class immediately tries to open *all* of the files simultaneously a few lines below:

https://github.com/cta-sst-1m/protozfitsreader/blob/c7d5f75a951ae1822c930d4b886598d6f5a52387/protozfits/__init__.py#L221-L222

So if such a problem as you have in mind occurs one will see this immediately and not after some possibly long time.

Reply:
However, I just realize, that you probably do not know what `MultiZFitsFiles` is intended for:

So in LST they have multiple file writers, which dump the incoming events into different files, so in case one has 10 events and 4 files, it might look like this:
```
f1 = [0, 4]
f2 = [1, 5, 8]
f3 = [2, 6, 9]
f4 = [3, 7]
```

The task of `MultiZFitsFiles` is to open these 4 files simultaneously and return the events in the correct order, so the user does not really have to know about these existence of 4 files.

I think this explanation should go into the `__doc__` of MultiZFitsFiles.

Reply:
Ah yes you are right. Did not get that it opened all of them at first. Thanks